dockstore-webservice/src/main/java/io/dockstore/webservice/resources/proposedGA4GH/ToolsExtendedApi.java

```diff
@@ -275,6 +302,17 @@ private static final class AggregatedMetricsPut {
         public static final String UNAUTHORIZED_RESPONSE = "Credentials not provided or incorrect.";
     }
 
+    private static final class AggregatedMetricsGet {
+        public static final String SUMMARY = "Get aggregated execution metrics for a workflow that was executed on a platform.";
```

Comment:
Here and the next line you have `workflow`; the lines below say `tool`. Given that this is TRS (extended), you should probably stick with `tool`.